cmake/README.md

```diff
@@ -0,0 +1,45 @@
+# CMake Modules Provided by ESMF
+
+The ESMF team provides CMake[1] modules that are useful when ESMF is needed
+within a CMake based build system. These modules can find libraries, create
+targets, and set variables within the scope of a CMake build. They may be
+used as is or customized for application specific build systems.
```

Comment:
Could we add a sentence at the end of this paragraph to point out that CMake module files are typically kept with the project that use them? Maybe:
"Typically CMake module files are copied into the project that use them. They become part of the project and are maintained with it."